apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/pom.xml

```diff
@@ -16,6 +16,7 @@
     </description>
 
     <properties>
+        <checkstyle.skip>false</checkstyle.skip>
```

Comment:
Can take this out, parent pom's skip has been removed and merged.

Reply:
taken out